app/views/marketing_redesign/about/show.html.erb

```diff
@@ -1,3 +1,19 @@
 <%= content_for(:page_title, t(".page_title")) %>
+<section class="section about-us">
+  <div class="image-crop">
+    <%= image_tag "marketing_redesign/man-learning-from-home.jpg", alt: "Man wearing a blue cap and green jacket sits at a table and uses a laptop, with a red mug in front of him in a well-lit room."%>
```

Comment:
Maybe this is nitpicky, but is the person's gender relevant? If not, maybe we should avoid specifying it. (If so [maybe we're trying to be transparent about our demographics/diversity], it seems probably equally relevant to mention their race.)

```suggestion
    <%= image_tag "marketing_redesign/person-learning-from-home.jpg", alt: "Person wearing a blue cap and green jacket sits at a table and uses a laptop, with a red mug in front of them in a well-lit room."%>
```

(I also don't notice anything in the photo that definitively suggests they're a man; I know a half-dozen people who have beards & short hair and aren't men. 🩷)